dev-tools/mage/crossbuild.go

```diff
@@ -43,8 +43,8 @@ const defaultCrossBuildTarget = "golangCrossBuild"
 // See NewPlatformList for details about platform filtering expressions.
 var Platforms = BuildPlatforms.Defaults()
 
-// Types is the list of package types
-var SelectedPackageTypes []PackageType
+// SelectedPackageTypes is the list of package types
+var SelectedPackageTypes []PackageType = []PackageType{TarGz}
```

Comment:
Is this correct? The description mentioned comments only, but this is code?

I see failures in the packaging

Reply:
![image](https://user-images.githubusercontent.com/2871786/151259878-968f13a7-1932-4639-a463-acf1bf29d7f5.png)

Reply:
This was a mistake, but it's been fixed already by #30007.